lib/cards/KCardGrid.vue

```diff
@@ -77,6 +136,47 @@
           return [LAYOUT_1_1_1, LAYOUT_1_2_2, LAYOUT_1_2_3].includes(value);
         },
       },
+      // eslint-enable-next-line kolibri/vue-no-unused-properties
+      /**
+       * Overrides the base grid `layout` for chosen breakpoints levels
+       */
+      // eslint-disable-next-line kolibri/vue-no-unused-properties
+      layoutOverride: {
+        type: Array,
+        required: false,
+        default: null,
+      },
+      // eslint-enable-next-line kolibri/vue-no-unused-properties
+      /**
+       * Set to `true` as long as data for cards
+       * are being loaded to display loading skeletons
+       */
+      // eslint-disable-next-line kolibri/vue-no-unused-properties
+      loading: {
+        type: Boolean,
+        default: false,
+      },
+      // eslint-enable-next-line kolibri/vue-no-unused-properties
+      /**
+       * Configures loading skeletons
+       */
+      // eslint-disable-next-line kolibri/vue-no-unused-properties
+      skeletonsConfig: {
```

Comment:
Hopefully upgrading to newer versions of eslint eventually will prevent us having to do this so much?

Reply:
That'd be nice